src/commands/info.rs

```diff
@@ -82,7 +78,7 @@ pub fn execute(matches: &ArgMatches<'_>) -> Result<(), Error> {
     let errors =
         project.is_none() || org.is_none() || config.get_auth().is_none() || info_rv.is_err();
 
-    if !matches.is_present("quiet") {
+    if !quiet_mode() {
```

Comment:
Why do we have quiet mode and the debug logger? Shouldn't just debug logging with different levels be enough here? So then we just have a single option to configure?

Reply:
We had a discussion about it on slack. Basically, all logging (including progress bars) go to `stderr`, where CLI output goes to `stdout`. The logging level only controls the verbosity of the logging, but it will not prevent anything from being printed to `stdout`. This way we can easily pipe the output to other places, while still allowing some commands like `sentry-cli info`, `sentry-cli releases info <name>` to have useful exit codes, and make them work as a conditional guard in shell scripts, without polluting the `stdout`.